src/analysis/types.js

```diff
@@ -15,4 +15,7 @@ export type NodeType = {|
   +pluralName: string,
   +prefix: NodeAddressT,
   +defaultWeight: number,
+  // The '+description' property should be a several word description that makes
```

Comment:
nit: we usually use `` `backticks` `` to delimit code segments, and
the “`+`” isn’t part of the field name (it’s an immutability modifier),
so: “``The `description` property``…”

Also, using the phrase “human-readable string” will help readers more
quickly understand what you mean here.